cypress/integration/tests/upgrade/create_test_data.ts

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    hasToBeSkipped,
+    login,
+    preservecookies,
+} from "../../../utils/utils";
+import { TagType } from "../../models/developer/controls/tagtypes";
+import * as data from "../../../utils/data_utils";
+import { Tag } from "../../models/developer/controls/tags";
+import { CredentialType, SEC, UserCredentials } from "../../types/constants";
+import { Stakeholders } from "../../models/developer/controls/stakeholders";
+import { Jobfunctions } from "../../models/developer/controls/jobfunctions";
+import { BusinessServices } from "../../models/developer/controls/businessservices";
+import { Stakeholdergroups } from "../../models/developer/controls/stakeholdergroups";
+import { CredentialsSourceControlUsername } from "../../models/administrator/credentials/credentialsSourceControlUsername";
+import { getRandomCredentialsData } from "../../../utils/data_utils";
+import { Analysis } from "../../models/developer/applicationinventory/analysis";
+import { UpgradeData } from "../../types/types";
+import { CredentialsMaven } from "../../models/administrator/credentials/credentialsMaven";
+
+describe("Creating pre-requisites before an upgrade", { tags: "@upgrade" }, () => {
```

Comment:
Why do we have to create data in 6.0.1 ?

It is only required in 6.0.0 branch right ?

Reply:
No, this is required for upgrade from 6.0.1 to future versions tests